### PR TITLE
P17a deployments

### PR DESCRIPTION
**Pull request by cdobs**
Creates ingestion csvs for P17a glider deployments

### CODE REVIEW

File: CP05MOAS-G0913/CP05MOAS-G0913_D00002_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-G0913/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-G0913-00-ENG000000,telemetered,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-G0913/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-G0913-02-FLORTM000,telemetered,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-G0913/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-G0913-03-CTDGVM000,telemetered,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-G0913/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-G0913-04-DOSTAM000,telemetered,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-G0913/D00002/merged-from-glider/cp_*.mrg,CP05MOAS-G0913-05-PARADM000,telemetered,Not Expected,short deployment due to iridium issues
```
Comment (rtravis-whoi) [inline CP05MOAS-G0913/CP05MOAS-G0913_D00002_ingest.csv:6]: looks good

File: CP05MOAS-G0913/CP05MOAS-G0913_R00002_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+#mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-G0913/R00002/dvl/*.PD0,CP05MOAS-G0913-01-ADCPAM000,recovered_host,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-G0913/R00002/merged/cp_*.mrg,CP05MOAS-G0913-02-FLORTM000,recovered_host,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-G0913/R00002/merged/cp_*.mrg,CP05MOAS-G0913-03-CTDGVM000,recovered_host,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-G0913/R00002/merged/cp_*.mrg,CP05MOAS-G0913-04-DOSTAM000,recovered_host,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-G0913/R00002/merged/cp_*.mrg,CP05MOAS-G0913-05-PARADM000,recovered_host,Not Expected,short deployment due to iridium issues
```
Comment (rtravis-whoi) [inline CP05MOAS-G0913/CP05MOAS-G0913_R00002_ingest.csv:7]: looks good

File: CP05MOAS-GL380/CP05MOAS-GL380_D00011_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-05-PARADM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP05MOAS-GL380/CP05MOAS-GL380_D00011_ingest.csv:6]: looks good - data available.

File: CP05MOAS-GL514/CP05MOAS-GL514_D00003_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/D00003/merged-from-glider/cp_*.mrg,CP05MOAS-GL514-00-ENG000000,telemetered,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/D00003/merged-from-glider/cp_*.mrg,CP05MOAS-GL514-02-FLORTM000,telemetered,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/D00003/merged-from-glider/cp_*.mrg,CP05MOAS-GL514-03-CTDGVM000,telemetered,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/D00003/merged-from-glider/cp_*.mrg,CP05MOAS-GL514-04-DOSTAM000,telemetered,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/D00003/merged-from-glider/cp_*.mrg,CP05MOAS-GL514-05-PARADM000,telemetered,Not Expected,short deployment due to iridium issues
```
Comment (rtravis-whoi) [inline CP05MOAS-GL514/CP05MOAS-GL514_D00003_ingest.csv:6]: looks good

File: CP05MOAS-GL514/CP05MOAS-GL514_R00003_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+#mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00003/dvl/*.PD0,CP05MOAS-GL514-01-ADCPAM000,recovered_host,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00003/merged/cp_*.mrg,CP05MOAS-GL514-02-FLORTM000,recovered_host,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00003/merged/cp_*.mrg,CP05MOAS-GL514-03-CTDGVM000,recovered_host,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00003/merged/cp_*.mrg,CP05MOAS-GL514-04-DOSTAM000,recovered_host,Not Expected,short deployment due to iridium issues
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL514/R00003/merged/cp_*.mrg,CP05MOAS-GL514-05-PARADM000,recovered_host,Not Expected,short deployment due to iridium issues
```
Comment (rtravis-whoi) [inline CP05MOAS-GL514/CP05MOAS-GL514_R00003_ingest.csv:7]: looks good

**Review by rtravis-whoi — APPROVED** [review]
looks good. What about the other recovered gliders 339, 376, 379?

File: CP05MOAS-G0913/CP05MOAS-G0913_D00002_ingest.csv (hunk 1 above)
Comment (pbrickley) [reply CP05MOAS-G0913/CP05MOAS-G0913_D00002_ingest.csv:6]: ok

File: CP05MOAS-G0913/CP05MOAS-G0913_R00002_ingest.csv (hunk 2 above)
Comment (pbrickley) [reply CP05MOAS-G0913/CP05MOAS-G0913_R00002_ingest.csv:7]: ok

File: CP05MOAS-GL380/CP05MOAS-GL380_D00011_ingest.csv (hunk 3 above)
Comment (pbrickley) [reply CP05MOAS-GL380/CP05MOAS-GL380_D00011_ingest.csv:6]: perfect

File: CP05MOAS-GL514/CP05MOAS-GL514_D00003_ingest.csv (hunk 4 above)
Comment (pbrickley) [reply CP05MOAS-GL514/CP05MOAS-GL514_D00003_ingest.csv:6]: ok

File: CP05MOAS-GL514/CP05MOAS-GL514_R00003_ingest.csv (hunk 5 above)
Comment (pbrickley) [reply CP05MOAS-GL514/CP05MOAS-GL514_R00003_ingest.csv:7]: ok

**Comment by pbrickley** [comment]
OK

**Review by pbrickley — APPROVED** [review]

**Pull request merged**